app/src/features/detail/components/PackageSidebar.tsx

```diff
@@ -191,8 +191,23 @@ const PackageSidebar = ({ data, loading, error }: PackageSidebarProps) => {
               <DescriptionIcon fontSize="small" style={{ marginRight: 6 }} />
               {data.documentation}
             </Link>
-          </div>
-        )}
+          ) : data.docsIpfsUrl ? (
+            <Link
+              href={`/docs/${data.name}/${data.version}`}
```

Comment:
nit: if package names/versions contain characters like spaces, slashes, or “@”, the current href can break routing.
Also, for an internal route you’ll get better UX and prefetch by using `NextLink`.
```suggestion
            <Link
              component={NextLink}
              href={`/docs/${encodeURIComponent(data.name)}/${encodeURIComponent(data.version)}`}
```